externalGravity.cpp

```diff
@@ -42,6 +44,10 @@ void ExternalGravity::AddParams(PRM prm)
     dJ6 = 0.0;
     prmAddParam(prm,"dJ6",paramDouble,&dJ6,
                 sizeof(double),"dJ6","<Oblateness coefficient J6 of central body> = 0.0");
+                
+    bLogarithmicHalo = 0;
```

Comment:
The compile fails here; I think you forgot to edit externalGravity.h to define dLogarithmicHalo